.projections.json

```diff
@@ -0,0 +1 @@
+/Users/jfrerich/dotfiles/mattermost/.projections.json
```

Comment:
Unrelated change?

Reply:
removed.  That's actually a handy file that makes opening alternative files easy. The most common example is defining test files as alternative files.  https://github.com/jfrerich/dotfiles/blob/master/mattermost/.projections.json